### ISSUE TITLE
add accessibility help dialog for debug console

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#221603]
Some things it should include:

- [x] An explanation to use arrow keys from input box to reach debug output
- [x] The setting `debug.expandLazyVariables`
- [x] Kbs for stepping over,  into, out of, and continue

The following should be added to the editor's accessibility help dialog:
- [x] Kb to set a breakpoint in the editor
- [x] The command which allows editor selection to be evaluated in the debug console, `editor.debug.action.selectionToRepl`, no keybinding assigned yet, maybe needs one

cc @jooyoungseo , @rperez030
